pkg/providers/vm_provider_interface.go

```diff
@@ -22,7 +22,8 @@ type VirtualMachineProviderInterface interface {
 	PublishVirtualMachine(ctx context.Context, vm *vmopv1.VirtualMachine,
 		vmPub *vmopv1.VirtualMachinePublishRequest, cl *imgregv1a1.ContentLibrary, actID string) (string, error)
 	GetVirtualMachineGuestHeartbeat(ctx context.Context, vm *vmopv1.VirtualMachine) (vmopv1.GuestHeartbeatStatus, error)
-	GetVirtualMachineGuestInfo(ctx context.Context, vm *vmopv1.VirtualMachine) (map[string]string, error)
+	// GetVirtualMachineGuestInfo(ctx context.Context, vm *vmopv1.VirtualMachine) (map[string]string, error)
```

Comment:
Why keep this around?

Reply:
I was not necessarily going to do so. I just have not removed it yet. Or rather I forgot to remove it. I'll do so now. Thanks!